src/caret_analyze/plot/timeseries/response_time_timeseries.py

```diff
@@ -111,5 +111,7 @@ def to_timeseries_records_list(
                 response_timeseries_list.append(response.to_worst_case_records())
             elif self._case == 'all':
                 response_timeseries_list.append(response.to_all_records())
+            elif self._case == 'worst_in_input':
```

Comment:
```suggestion
            elif self._case == 'worst-in-input':
```

Reply:
Fixed in https://github.com/tier4/CARET_analyze/pull/336/commits/318c038007dfb0f14bd6648fe8dedc6824680d2a.